### PR TITLE
Show how to add Fastlane plugins

### LINKED ISSUE #447: document how to install fastlane plugins on build machine
One user reported that this script failed:

```
      - name: Install fastlane-plugin-browserstack
        script: |
          gem install bundler
          sudo gem install fastlane-plugin-browserstack --user-install
```
Does not seem to work, getting this error:

```
Could not find action, lane or variable 'upload_to_browserstack_app_live'
```

Let's try to validate and document it. He says solution was to do this

```
      - name: Install fastlane-plugin-browserstack
        script: |
          cd ios
          bundle add fastlane-plugin-browserstack
```

ref: https://codemagicio.slack.com/archives/CEKE2KZ37/p1629143385271800

### PR DESCRIPTION
**Pull request by nevercode-kevin**
resolves #447

### CODE REVIEW

File: content/integrations/fastlane-integration.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ gem "fastlane"
 gem "cocoapods"
 ```
 
+## Fastlane plugins
+
+If you are using any Fastlane plugins you should create a script to install them as follows:
```
Comment (helinanever) [inline content/integrations/fastlane-integration.md:45]:
```suggestion
If you are using any Fastlane plugins, you should create a script to install them as follows:
```

File: content/integrations/fastlane-integration.md [hunk 2]
```diff
@@ -40,6 +40,18 @@ gem "fastlane"
...
+
+```
+      - name: Install Fastlane plugins
+        script: |
+          cd ios     # change to ios folder if required
```
Comment (helinanever) [inline content/integrations/fastlane-integration.md:50]: should the comment be about android folder?

File: content/integrations/fastlane-integration.md [hunk 3]
```diff
@@ -40,6 +40,18 @@ gem "fastlane"
...
 ```
 
+## Fastlane plugins
+
+If you are using any Fastlane plugins, you should create a script to install them as follows:
+
+```
+      - name: Install Fastlane plugins
+        script: |
+          cd ios     # change to ios folder if required
```
Comment (nevercode-kevin) [inline content/integrations/fastlane-integration.md:50]:
```suggestion
          cd ios     # change to ios/android folder as required
```

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**